tests/unit/test_pipelines.py

```diff
@@ -61,7 +61,7 @@ def expected_response():
 
 
 def test_prepare_payload(
-    tmpdir, json_spider_record, spider, expected_response,
+    tmpdir, json_spider_record, spider, expected_response, set_up_clean_up_env_vars,
```

Comment:
As you are using the `autouse` parameter, you don't need to explicitly put in in the prameters, this is not needed, it will be used by default, everywhere ;)